src/warmer.js

```diff
@@ -186,15 +190,15 @@ module.exports.warmUp = async (event, context) => {
 
     try {
       await Promise.all(Array(concurrency).fill(0).map(async () => await lambda.invoke(params).promise()));
-      console.log(\`Warm Up Invoke Success: \${func.name}\`);
+      logger(\`Warm Up Invoke Success: \${func.name}\`);
       return true;
     } catch (e) {
       console.log(\`Warm Up Invoke Error: \${func.name}\`, e);
```

Comment:
Please change this to `console.error(...)` to be more clear

Reply:
The `console.error` approach is resulting in a bunch of warnings when running the unit tests. Is that ok?

![image](https://user-images.githubusercontent.com/64426814/152269911-92480b99-9d0c-4db8-b2da-7faca9aafdc5.png)